.github/workflows/test-go.yaml

```diff
@@ -144,6 +155,10 @@ jobs:
         fi
         GO_FAIL_SUMMARY=$GO_FAIL_SUMMARY envsubst < .github/workflows/config/slack_payload_template.json > ./payload.json
 
+    # TODO: figure out a sane way to combine outputs from different matrix jobs
+    # into a single slack notification, instead of sending one per job. This
+    # problem already existed but now it's accentuated because we're running 4
+    # jobs.
```

Comment:
I didn't want to spend more than ~1 hour on this so it seemed like an acceptable tradeoff, but raising in case somebody disagrees.

Reply:
Let's start like this and iterate if it becomes too noisy.